ios/MullvadVPN/View controllers/AccountDeletion/AccountDeletionContentView.swift

```diff
@@ -326,7 +335,7 @@ class AccountDeletionContentView: UIView {
         } else {
             activityIndicator.stopAnimating()
         }
-        deleteButton.isEnabled = isDeleteButtonEnabled && isAccountNumberLengthSatisfied
+        deleteButton.isEnabled = isDeleteButtonEnabled && isInputValid
```

Comment:
Should `isDeleteButtonEnabled` be updated too?

Currently it still says the button should be enabled upon failure:
https://github.com/mullvad/mullvadvpn-app/blob/9cc7a43a5dd76ccae541a43a7b95ffe61e627898/ios/MullvadVPN/View%20controllers/AccountDeletion/AccountDeletionContentView.swift#L221-L228

Reply:
We want to disable the button when loading (to avoid interaction while trying to delete the account), otherwise enabled.